nasl/nasl_cert.c

```diff
@@ -761,10 +761,15 @@ get_name (const char *string)
  *
  * - @a image       Return the entire certificate as binary data.
  *
- * - @a algorithm-name: Return the algorithm name. Get the OID of
- *                      the digest algorithm and translated to a name
- *                      from a list from Wireshark.
- *                      See epan/dissectors/packet-pkcs1.c
+ * - @a algorithm-name: Same as signature-algorithm-name. TODO: Remove it and
+ *                      and leave only signature-algorithm-name.
+ *
+ * - @a signature-algorithm-name: Return the algorithm name. Get the OID of
```

Comment:
```suggestion
 * - @a signature-algorithm-name: Return the algorithm name used to sign the certificate. Get the OID of
```

Not absolutely sure about the text but this should make it a little bit more clear.